R/use_r_workflows.R

```diff
@@ -82,32 +82,62 @@ use_r_cmd_check <- function(workflow_name = "call-r-cmd-check.yml",
   invisible(workflow_name)
 }
 
-#' workflow for calculating code coverage
+#' Workflow for calculating coverage and summarizing using octocov toolkit
+#' This workflow calculates coverage using the covr package, then creates 
+#' summaries to post as a GitHub action summary or as pull request comment. All
+#' data remains in the GitHub repository.
 #' @template workflow_name
-#' @param use_public_rspm Use posit package manager instead of CRAN to install dependencies? The
-#'  advantage here is that dependencies are precompiled, so install should be much quicker. In
-#'  rare situations (like packages with TMB dependencies), using use_public_rspm = FALSE may be
-#'  a better option. Note a setting only needs to be specified in the yml if use_public_rspm is FALSE, so
-#'  there will be no setting added if use_public_rspm is TRUE.
+#' @template use_public_rspm
+#' @export
+use_calc_cov_summaries <- function(workflow_name = "call-calc-cov-summaries.yml", use_public_rspm = TRUE) {
+  check_workflow_name(workflow_name)
+  usethis::use_github_action("call-calc-cov-summaries.yml",
+    save_as = workflow_name,
+    url = "https://raw.githubusercontent.com/nmfs-fish-tools/ghactions4r/main/inst/templates/call-calc-cov-summaries.yml"
+  )
+  path_to_yml <- file.path(".github", "workflows", workflow_name)
+  gha <- readLines(path_to_yml)
+  if (use_public_rspm == FALSE) {
+    path_to_yml <- file.path(".github", "workflows", workflow_name)

```

Comment:
Lines 98-99 and lines 101-102 are identical. Consider keeping just one?

Reply:
good catch, thank you!